WalletWasabi.Fluent/ViewModels/Wallets/Coins/CoinViewModel.cs

```diff
@@ -26,10 +25,14 @@ public CoinViewModel(LabelsArray labels, ICoinModel coin, bool ignorePrivacyMode
 		IsSelected = false;
 		ScriptType = coin.ScriptType;
 		IgnorePrivacyMode = ignorePrivacyMode;
+		this.WhenAnyValue(x => x.Coin.IsExcludedFromCoinJoin).BindTo(this, x => x.IsExcludedFromCoinJoin).DisposeWith(_disposables);
 		this.WhenAnyValue(x => x.Coin.IsCoinJoinInProgress).BindTo(this, x => x.IsCoinjoining).DisposeWith(_disposables);
+		this.WhenAnyValue(x => x.Coin.IsCoinJoinInProgress, b => !b).BindTo(this, x => x.CanBeSelected).DisposeWith(_disposables);
```

Comment:
This should be configurable! The Condition you have here should only apply when the user on the Exclude coins dialog.

https://github.com/zkSNACKs/WalletWasabi/pull/12893#issuecomment-2069284026